tackle-test-generator-unit/pom.xml

```diff
@@ -89,19 +82,15 @@
       <version>[4.5.13,)</version>
     </dependency>
     <dependency>
-      <groupId>org.evosuite</groupId>
-	  <artifactId>evosuite</artifactId>
+      <groupId>com.github.EvoSuite.evosuite</groupId>
+      <artifactId>evosuite-master</artifactId>
```

Comment:
why do we need both evosuite and evosuite-master?

Reply:
Yes, I know; that's what's available on jitpack (I spent a lot of time figuring this out)